diagnostic.md

```diff
@@ -150,7 +174,10 @@ div {
 ```
 
 <!-- your answer starts here -->
-
+divOne is floated to the right
+divTwo and divThree are floated to the left
+divFour is on separate line
+divFive is floated to the left
```

Comment:
.divOne will be in the top-right corner; .divTwo will be in the top-left corner, with .divThree immediately to its right; .divFour will be underneath .divTwo, and .divFive will sit immediately to the right of .divFour (under .divThree).

In short: 2 3 1 4 5